File: DentOffice.Model/MedicinskiKarton.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DentOffice.Model
{
    public class MedicinskiKarton
    {
        public int MedicinskiKartonID { get; set; }
        public DateTime Datum { get; set; }
        public string Napomena { get; set; }
    }
}

File: dentofficemobile/lib/models/medicinski_karton.dart
// -- medicinski_karton.dart --
import 'package:json_annotation/json_annotation.dart';

part 'medicinski_karton.g.dart';

@JsonSerializable()
class MedicinskiKarton {
  int medicinskiKartonID;
  DateTime datum;
  String napomena;

  MedicinskiKarton({required this.medicinskiKartonID,required this.datum,required this.napomena,});

  factory MedicinskiKarton.fromJson(Map<String, dynamic> json) => _$MedicinskiKartonFromJson(json);

  Map<String, dynamic> toJson() => _$MedicinskiKartonToJson(this);
}